### PR TITLE
Make SEPA line1 optional

### PR DESCRIPTION
**Pull request by bg-stripe**
r? @bdorfman-stripe

### CODE REVIEW

File: Stripe/STPSourceParams.m [hunk 1]
```diff
@@ -150,17 +150,16 @@ + (STPSourceParams *)sepaDebitParamsWithName:(NSString *)name
...
-      @"city": city,
-      @"postal_code": postalCode,
-      @"country": country
-      };
-
+    NSMutableDictionary<NSString *,NSString *> *address = [NSMutableDictionary new];
+    address[@"city"] = city;
+    address[@"postal_code"] = postalCode,
+    address[@"country"] = country;
+    if (addressLine1) {
```
Comment (bdorfman-stripe) [inline Stripe/STPSourceParams.m]: Is this check even necessary? Setting a nil value with a mutable dictionary is valid iirc.

**Comment by bdorfman-stripe** [comment]
LGTM.

Should we have some tests to verify optionality stuff is working correctly? I guess we are somewhat just enforcing it through nullability annotations so not sure if testable without compiler errors in our setup.

File: Stripe/STPSourceParams.m (hunk 1 above)
Comment (bg-stripe) [inline Stripe/STPSourceParams.m]: oh whoops, good point.

Comment (bg-stripe) [inline Stripe/STPSourceParams.m]: (updated)

**Comment by bdorfman-stripe** [comment]
👍

**Pull request merged**